packages/components/src/SquigglePlayground.tsx

```diff
@@ -0,0 +1,129 @@
+import React, { FC, useState } from "react";
+import ReactDOM from "react-dom";
+import { SquiggleChart } from "./SquiggleChart";
+import CodeEditor from "./CodeEditor";
+import { Form, Input, Card, Row, Col } from "antd";
+import "antd/dist/antd.css";
+
+interface FieldFloatProps {
+  label: string;
+  className?: string;
+  value: number;
+  onChange: (value: number) => void;
+}
+
+function FieldFloat(Props: FieldFloatProps) {
+  let [contents, setContents] = useState(Props.value + "");
+  return (
+    <Form.Item label={Props.label}>
+      <Input
+        value={contents}
+        className={Props.className ? Props.className : ""}
+        onChange={(e) => setContents(e.target.value)}
```

Comment:
My impression is that this doesn't run a Props.onChange() when on the onChange() handler is called, but only when the onBlur() is called. This doesn't seem right to me.

If you don't want every key stroke to change things, because of many key strokes, then I'd probably add a debouncer, instead of just having updates on ``onBlur``.

Reply:
It seems like this still hasn't been addressed? This one seems fairly important to me.

Reply:
Ok, I've decided to throw out onBlur for onChange.